packages/synapse-react-client/src/components/widgets/facet-nav/SelectionCriteriaPills.tsx

```diff
@@ -35,10 +35,13 @@ function getPillPropsFromColumnQueryFilter(
   queryVisualizationContext: QueryVisualizationContextType,
 ): SelectionCriteriaPillProps[] {
   const { getColumnDisplayName } = queryVisualizationContext
-  const columnModel = queryContext.getColumnModel(queryFilter.columnName)!
+  const columnModel = queryContext.getColumnModel(queryFilter.columnName)
   // ColumnSingleValueQueryFilter and ColumnMultiValueQueryFilter both allow for a list of values
   // If there are more than _n_ values, consolidate to one pill
-  if (queryFilter.values.length > MAX_VALUES_IN_FILTER_FOR_INDIVIDUAL_PILLS) {
+  if (
+    queryFilter.values.length > MAX_VALUES_IN_FILTER_FOR_INDIVIDUAL_PILLS ||
+    !columnModel
```

Comment:
We don't have a column model because the filter is applied to a column on the table referenced in `definingSql`. 

For now it will look like this, I will open a new ticket to improve this:

![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/0dd984ae-a244-40b8-9bbe-0fbd6e8dc521)

Reply:
Ticket is intentionally vague because it's not clear to me how we want to approach this w.r.t. the design: https://sagebionetworks.jira.com/browse/PORTALS-2775